### ISSUE TITLE
Question about whether to use multiview feature

### ISSUE THREAD

**Issue by WeitaiKang** [ZzZZCHS/WS-3DVG#4]
Dear authors, I want to ask whether you were using multiview feature from enet.

As I have found, the code guideline of yours provide the preparation of multiview feature from enet, and the config sets [use_multiview](https://github.com/ZzZZCHS/WS-3DVG/blob/ff14e630456cce1d2520088e5b1246a25909ab66/config/config.py#L80) as True as default. However, the [init of model](https://github.com/ZzZZCHS/WS-3DVG/blob/ff14e630456cce1d2520088e5b1246a25909ab66/scripts/joint_scripts/train.py#L91) doesn't involve multiview feature (is it the input_channels=0?), and the code of [GroupFree](https://github.com/zeliu98/Group-Free-3D/blob/master/scannet/scannet_detection_dataset.py) doesn't seem to load multiview features. Since you were using the checkpoint from GroupFree, I am confused whether you were really using multiview feature. If not, then we don't bother to prepare it at the first place.

I also wander if you were using [normal](https://github.com/ZzZZCHS/WS-3DVG/blob/ff14e630456cce1d2520088e5b1246a25909ab66/lib/joint/dataset.py#L694), where your config sets it as True as default but again is not used in [GroupFree](https://github.com/zeliu98/Group-Free-3D/blob/master/scannet/scannet_detection_dataset.py) .

**Comment by vice-jin** [comment]
Hello, when I was reproducing their paper, the accuracy at IoU 0.25 was only 6%. Is this consistent with your reproduction results? I have emailed them, but they have not replied.

**Comment by WeitaiKang** [comment]
Oh really? I haven't reproduced their method.

**Comment by ZzZZCHS** [comment]
> Dear authors, I want to ask whether you were using multiview feature from enet.
> 
> As I have found, the code guideline of yours provide the preparation of multiview feature from enet, and the config sets [use_multiview](https://github.com/ZzZZCHS/WS-3DVG/blob/ff14e630456cce1d2520088e5b1246a25909ab66/config/config.py#L80) as True as default. However, the [init of model](https://github.com/ZzZZCHS/WS-3DVG/blob/ff14e630456cce1d2520088e5b1246a25909ab66/scripts/joint_scripts/train.py#L91) doesn't involve multiview feature (is it the input_channels=0?), and the code of [GroupFree](https://github.com/zeliu98/Group-Free-3D/blob/master/scannet/scannet_detection_dataset.py) doesn't seem to load multiview features. Since you were using the checkpoint from GroupFree, I am confused whether you were really using multiview feature. If not, then we don't bother to prepare it at the first place.
> 
> I also wander if you were using [normal](https://github.com/ZzZZCHS/WS-3DVG/blob/ff14e630456cce1d2520088e5b1246a25909ab66/lib/joint/dataset.py#L694), where your config sets it as True as default but again is not used in [GroupFree](https://github.com/zeliu98/Group-Free-3D/blob/master/scannet/scannet_detection_dataset.py) .

We only used xyz of point clouds as the input. So you can skip the processing of multiview features and set use_multiview to False. The use_normal can also be set to False. (It was a mistake that we still set them to True in the default setting, which made the dataloader load related features that we actually didn't use.)

**Comment by ZzZZCHS** [comment]
> Hello, when I was reproducing their paper, the accuracy at IoU 0.25 was only 6%. Is this consistent with your reproduction results? I have emailed them, but they have not replied.

Hi, I'm sorry that we may have missed your email. Is there any detailed information about your reproduction? You can send it to my email at huanghaifeng@zju.edu.cn or huanghaifeng317@gmail.com, or open an issue.

**Comment by WeitaiKang** [comment]
Great, thanks for your clarification!

**Comment by WeitaiKang** [comment]
I have an additional question. So you are saying you just use xyz. Then should I use [height](https://github.com/ZzZZCHS/WS-3DVG/blob/ff14e630456cce1d2520088e5b1246a25909ab66/config/config.py#L68)?

**Comment by ZzZZCHS** [comment]
We didn't use height. But you can try it.

**Comment by WeitaiKang** [comment]
Ok thank you.

**Comment by Liuuuuuyh** [comment]
> Hello, when I was reproducing their paper, the accuracy at IoU 0.25 was only 6%. Is this consistent with your reproduction results? I have emailed them, but they have not replied.

Hello! I obtained the same results as you did. Have you been successful in reproducing the results mentioned in the research paper?

**Comment by ZzZZCHS** [comment]
> > Hello, when I was reproducing their paper, the accuracy at IoU 0.25 was only 6%. Is this consistent with your reproduction results? I have emailed them, but they have not replied.
> 
> Hello! I obtained the same results as you did. Have you been successful in reproducing the results mentioned in the research paper?

@Liuuuuuyh, I've solved his problem before. It was because he loaded a wrong GroupFree pretrained checkpoint. Maybe you can also check whether you've downloaded the right one.
![Screenshot 2024-04-26 at 10 29 33 PM (1)](https://github.com/ZzZZCHS/WS-3DVG/assets/50910784/170bb594-c814-43e7-a398-ce4204aa5c2c)
